src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -624,11 +640,22 @@ type Comment {
   """
   status: COMMENT_STATUS!
 
+  """
+  parentCount is the number of direct parents for this Comment. Currently this
+  value is the same as depth.
```

Comment:
When will this ever be different?

Reply:
If we decide to change the count based on deleted/rejected/tombstoned comments